doc/browsing_cubes/overlay.ipynb

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
I have not yet made all the changes that I need to in the notebooks.  Although I have added this one in here, I intend to make a new PR to tidy up and update (and add introductions and descriptions and things) to all the notebooks.